Rdmp.UI/SimpleDialogs/UserSettingsUI.resx

```diff
@@ -57,7 +57,7 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
-  <assembly alias="System.Drawing.Common" name="System.Drawing.Common, Version=5.0.0.2, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51" />
+  <assembly alias="System.Drawing.Common" name="System.Drawing.Common, Version=6.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51" />
```

Comment:
Hmn this line looks a bit sketch.  Hope its not going to come back to haunt us.  Looks like Visual Studio Designer shenannigans

Reply:
Since we are apparently using 5.0.0.2 NuGet package I think we don't want this change, reverted in d792989